src/app/frontend/replicationcontrollerdetail/replicationcontrollerdetail_controller.js

```diff
@@ -108,6 +106,18 @@ export default class ReplicationControllerDetailController {
      * @export {boolean}
      */
     this.eventsOrder = DOWNWARDS;
+
+    /**
+     * If true then sidebar will be displayed, otherwise it won't.
+     * @export {boolean}
+     */
+    this.displaySidebar = this.mdMedia('gt-sm');
+
+    /**
+     * Binding resize listener, which switches displaySidebar boolean variable.
+     */
+    angular.element(this.window)
```

Comment:
Why don't you prefer doing all those element and resize staff in the directive link function?